src/MinimalDelegation.sol

```diff
@@ -112,10 +115,16 @@ contract MinimalDelegation is IERC7821, ERC1271, EIP712, ERC4337Account, Receive
     /// @inheritdoc IERC4337Account
     function updateEntryPoint(address entryPoint) external {
         _onlyThis();
-        MinimalDelegationStorageLib.get().entryPoint = entryPoint;
+        MinimalDelegationStorageLib.get().entryPoint = EntrypointLib.pack(entryPoint);
```

Comment:
nit: but I prefer libraries are used like
`entryPoint.pack()`

Reply:
hmm yeah but we have different types here, the packed is 256 while we're wrapping an address here

Reply:
what do you mean? `entryPoint` is an address

so you can do `using EntrypointLib for address` no?